pycytominer/operations/transform.py

```diff
@@ -43,17 +45,25 @@ def __init__(self, epsilon=1e-6, center=True, method="ZCA"):
             option to center the input X matrix
         method : str, default "ZCA"
             a string indicating which class of sphering to perform
+        return_numpy: bool, default False
+            option to return ndarray, instead of dataframe
         """
         avail_methods = ["PCA", "ZCA", "PCA-cor", "ZCA-cor"]
 
         self.epsilon = epsilon
         self.center = center
+        self.return_numpy = return_numpy
 
-        assert (
-            method in avail_methods
-        ), f"Error {method} not supported. Select one of {avail_methods}"
+        if method not in avail_methods:
+            raise ValueError(
+                f"Error {method} not supported. Select one of {avail_methods}"
+            )
         self.method = method
 
+        # PCA-cor and ZCA-cor require center=True
```

Comment:
Consider adding a source/documentation for why this is required.

Reply:
Thanks for the nudge

Reply:
Saw the comment, well done